docs/design/bundle-sources.md

```diff
@@ -0,0 +1,300 @@
+
+<h3 id="prereqs-adding-a-plain-bundle-to-fbc">
+Prerequisites
+</h3>
+
+* `opm` CLI tool
+* A plain bundle image
+* A file-based catalog
+* Push access to an image registry, such as [Quay](https://quay.io)
+* Docker or Podman
+
+<h3 id="proc-adding-a-plain-bundle-to-fbc">
+Procedure
+</h3>
+
+1. Verify that your catalog's `index.json` or `index.yaml` file is similar to the following example:
+
+    *Example `<catalog_dir>/index.json` file*
+
+    ```json
+    {   
+        {
+         "schema": "olm.package",
+         "name": "<operator_name>",
+         "defaultChannel": "<default_channel_name>"
+        }    
+    }
+    ```
+
+1. To create an `olm.bundle` blob, edit your `index.json` or `index.yaml` file, similar to the following example:
+
+    *Example `<catalog_dir>/index.json` file*
+
+    ```json
+    {
+        "schema": "olm.bundle",
+        "name": "<bundle_name>",
+        "package": "<operator_name>",
+        "image": "quay.io/<image_repo>/plainbundle:test",
+        "properties": [
+            {
+                "type": "olm.package",
+                "value": {
+                "packageName": "<operator_name>",
+                "version": "0.0.3"
+                }
+            },
+            {
+                "type": "olm.bundle.mediatype",
+                "value": "plain+v0"
+            }
+      ]
+    }
+    ```
+
+1. To create an `olm.channel` blob, edit your `index.json` or `index.yaml` file, similar to the following example:
+
+    *Example `<catalog_dir>/index.json` file*
+
+    ```json
+    {
+        "schema": "olm.channel",
+        "name": "<default_channel_name>",
+        "package": "<operator_name>",
+        "entries": [
+            {
+                "name": "<bundle_name>"
+            }
+        ]
+    }
+    ```
+
+<h3 id="verify-adding-a-plain-bundle-to-fbc">
+Verification
+</h3>
+
+1. Open your `index.json` or `index.yaml` file and ensure it is similar to the following example:
+
+    *Example `index.json` file*
+
+    ```json
+    {
+        "schema": "olm.package",
+        "name": "testOperator",
+        "defaultChannel": "preview"
+    }
+    {
+        "schema": "olm.bundle",
+        "name": "plainbundle:test",
+        "package": "testOperator",
+        "image": "quay.io/rashmigottipati/plainbundle:test",
+        "properties": [
+            {
+                "type": "olm.package",
+                "value": {
+                "packageName": "testOperator",
+                "version": "0.0.3"
+                }
+            },
+            {
+                "type": "olm.bundle.mediatype",
+                "value": "plain+v0"
+            }
+        ]
+    }
+    {
+        "schema": "olm.channel",
+        "name": "preview",
+        "package": "testOperator",
+        "entries": [
+            {
+                "name": "plainbundle:test"
+            }
+        ]
+    }
+    ```
+
+1. Run the following command to validate your catalog:
+
+    ```sh
+    opm validate <catalog_directory>
```

Comment:
I _think_ I would suggest removing this step. `opm validate` was designed to validate an FBC based on OLMv0's fairly strict expectations. 

Long-term, this requires more thought, but for now, since `opm` does not have any OLMv1-specific logic, I think I would avoid using it for this purpose.